### PR TITLE
Reportback script

### PR DESCRIPTION
**Pull request by weerd**
Current progress on new Reportback interface.

@sbsmith86 

CC: @DFurnes @barryclark

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/js/app.js [hunk 1]
```diff
@@ -85,11 +85,14 @@ define("app", function (require) {
...
 
-    // Initialize Reportback js.
-    var $stepFour = $body.find("#prove");
 
-    if ($body.hasClass("node-type-campaign") && $stepFour.length > 0) {
-      new Reportback();
+    /**
+     * Initialize Reportback js.
+     */
+    var $reportback = $body.find("#reportback");
```
Comment (sbsmith86) [inline lib/themes/dosomething/paraneue_dosomething/js/app.js:92]: Nice!

**Comment by sbsmith86** [comment]
This looks good to me.

**Comment by weerd** [comment]
Coolio! :dancer:

File: lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php [hunk 2]
```diff
@@ -309,11 +309,10 @@
                         </div>
                       </div>
                       <img src="data:image/gif;base64,R0lGODlhCgAKAJEAAAAAAP////Ly8gAAACwAAAAACgAKAAACCJSPqcvtD2MrADs=" alt="">
-                      <!-- <img src="http://placeimg.com/300/300/animals" alt=""> -->
                       <input class="js-image-upload-beta" type="file" name="files[reportback_file]" size="60">
                     </div>
                   </div>
-
+                  <p>POOP!</p>
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php:315]: :hankey:

**Pull request merged**